From: =?utf-8?q?Bastien_ROUCARI=C3=88S?= <roucaries.bastien@gmail.com>
Date: Fri, 11 Aug 2017 10:20:20 +0200
Subject: Fix www/support.html

Forwarded: Yes
---
 www/support.html | 1 +
 1 file changed, 1 insertion(+)

diff --git a/www/support.html b/www/support.html
index 2676915..53772fb 100644
--- a/www/support.html
+++ b/www/support.html
@@ -176,3 +176,4 @@
   <script src="assets/magick.js" ></script>
   </body>
 </html>
+
