From: =?utf-8?q?Bastien_Roucari=C3=A8s?= <rouca@debian.org>
Date: Sun, 5 Nov 2023 14:58:03 +0000
Subject: Add missing link flags

Forwarded: https://github.com/ImageMagick/ImageMagick/issues/6850
---
 coders/Makefile.am | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/coders/Makefile.am b/coders/Makefile.am
index 179a94f..f5e84ee 100644
--- a/coders/Makefile.am
+++ b/coders/Makefile.am
@@ -709,7 +709,7 @@ coders_art_la_LIBADD       = $(MAGICKCORE_LIBS)
 coders_ashlar_la_SOURCES   = coders/ashlar.c
 coders_ashlar_la_CPPFLAGS  = $(MAGICK_CODER_CPPFLAGS)
 coders_ashlar_la_LDFLAGS   = $(MODULECOMMONFLAGS)
-coders_ashlar_la_LIBADD    = $(MAGICKCORE_LIBS)
+coders_ashlar_la_LIBADD    = $(MAGICKCORE_LIBS) $(MATH_LIBS)
 
 # AVS coder module
 coders_avs_la_SOURCES      = coders/avs.c
@@ -787,7 +787,7 @@ coders_cmyk_la_LIBADD      = $(MAGICKCORE_LIBS)
 coders_cube_la_SOURCES     = coders/cube.c
 coders_cube_la_CPPFLAGS    = $(MAGICK_CODER_CPPFLAGS)
 coders_cube_la_LDFLAGS     = $(MODULECOMMONFLAGS)
-coders_cube_la_LIBADD      = $(MAGICKCORE_LIBS) $(GOMP_LIBS)
+coders_cube_la_LIBADD      = $(MAGICKCORE_LIBS) $(GOMP_LIBS) $(MATH_LIBS)
 
 # CUT coder module
 coders_cut_la_SOURCES      = coders/cut.c
@@ -805,7 +805,7 @@ coders_dcm_la_LIBADD       = $(MAGICKCORE_LIBS) $(MATH_LIBS)
 coders_dds_la_SOURCES      = coders/dds.c
 coders_dds_la_CPPFLAGS     = $(MAGICK_CODER_CPPFLAGS)
 coders_dds_la_LDFLAGS      = $(MODULECOMMONFLAGS)
-coders_dds_la_LIBADD       = $(MAGICKCORE_LIBS)
+coders_dds_la_LIBADD       = $(MAGICKCORE_LIBS) $(MATH_LIBS)
 
 # DEBUG coder module
 coders_debug_la_SOURCES    = coders/debug.c
@@ -1021,7 +1021,7 @@ coders_jnx_la_LIBADD       = $(MAGICKCORE_LIBS)
 coders_json_la_SOURCES     = coders/json.c
 coders_json_la_CPPFLAGS    = $(MAGICK_CODER_CPPFLAGS)
 coders_json_la_LDFLAGS     = $(MODULECOMMONFLAGS)
-coders_json_la_LIBADD      = $(MAGICKCORE_LIBS)
+coders_json_la_LIBADD      = $(MAGICKCORE_LIBS) $(MATH_LIBS)
 
 # JXL coder module
 coders_jxl_la_SOURCES      = coders/jxl.c
@@ -1255,7 +1255,7 @@ coders_ps3_la_LIBADD       = $(MAGICKCORE_LIBS) $(JBIG_LIBS) $(JPEG_LIBS) $(LZMA
 coders_psd_la_SOURCES      = coders/psd.c coders/psd-private.h
 coders_psd_la_CPPFLAGS     = $(MAGICK_CODER_CPPFLAGS)
 coders_psd_la_LDFLAGS      = $(MODULECOMMONFLAGS)
-coders_psd_la_LIBADD       = $(MAGICKCORE_LIBS)
+coders_psd_la_LIBADD       = $(MAGICKCORE_LIBS) $(ZLIB_LIBS)
 
 # PWP coder module
 coders_pwp_la_SOURCES      = coders/pwp.c
@@ -1528,7 +1528,7 @@ coders_xwd_la_LIBADD       = $(MAGICKCORE_LIBS) $(X11_LIBS)
 coders_yaml_la_SOURCES     = coders/yaml.c
 coders_yaml_la_CPPFLAGS    = $(MAGICK_CODER_CPPFLAGS)
 coders_yaml_la_LDFLAGS     = $(MODULECOMMONFLAGS)
-coders_yaml_la_LIBADD      = $(MAGICKCORE_LIBS)
+coders_yaml_la_LIBADD      = $(MAGICKCORE_LIBS) $(MATH_LIBS)
 
 # YCBCR coder module
 coders_ycbcr_la_SOURCES    = coders/ycbcr.c
